Resources/Prototypes/Maps/Pools/deathmatch.yml

```
@@ -1,5 +1,4 @@
 - type: gameMapPool
   id: DeathMatchMapPool
-  maps:
-  - MeteorArena
-  - dm01-entryway
+  maps: # Fire edit - все стандартные карты удалены в целях оптимизации использования RAM
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)